app/Console/Commands/IndexPeopleToMeilisearchCommand.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Console\Commands;
+
+use App\Models\Person;
+use Illuminate\Console\Command;
+use Meilisearch\Client;
+
+class IndexPeopleToMeilisearchCommand extends Command
+{
+    protected $signature = 'index:people-to-meilisearch';
+
+    protected $description = 'Command description';
+
+    public function handle(): void
+    {
+        $client = new Client(config('scout.meilisearch.host'), config('scout.meilisearch.key'));
+        $index = $client->index('people');
+
+        $people = Person::all(['id', 'name', 'birthday', 'still']);
```

Comment:
It will be much quicker if you use json syntax in mysql, that's the main bottleneck when importing data.

Since you're not using scout, try https://github.com/HDInnovations/UNIT3D-Community-Edition/pull/3061

This should also be ran daily.

Reply:
Also, you could try experimenting with adding a covering index for these columns. That way when you do a full table scan, it can do a full scan over just the index instead of the table.

Reply:
To be done in separate PR once get baseline in prod on current.